test

```diff
@@ -41,6 +41,11 @@ TEST=${split[@]/#/${REPO_PATH}/}
 echo "Running tests..."
 go test -timeout 3m ${COVER} $@ ${TEST} --race
```

Comment:
The default timeout is 10m, so if anything we could tighten it up a lot. PRs accepted.